### ISSUE TITLE
Perfomance on 2019 dataset

### ISSUE THREAD

**Issue by bilal2vec** [menglin0320/wap-on-tensor#3]
Hi,

I ran your code on the new 2019 crohme dataset (https://www.cs.rit.edu/~crohme2019) for 100 epochs but the training loss plateaus at ~0.7 and the val exprate is stuck at 0. Do you use any special preprocessing or normalization of the 2013 training data?

Thanks

**Comment by menglin0320** [comment]
IDK, the images are from jianshu zhang's repo, they are generated from offline images so by default they should be already binarized. 
Check size of images and check if the images are binarized.
But I guess you should already moved to other repos since it's been so long...
